Event Gather - on manual run, from and to ISO dates are required, not optional

### Describe the Bug

When running the Event Gather action workflow manually, the from and to ISO dates are described as "Optional ISO formatted string for datetime," but those values are assumed to exist in `event-gather-pipeline.yml` in the "Gather and Process Requested Events - Manual" step.  Without, the action fails:

```bash
cd python/
run_cdp_event_gather event-gather-config.json \
    --from  \
    --to 
...
usage: run_cdp_event_gather [-h] [-f FROM_DT] [-t TO_DT] config_file
run_cdp_event_gather: error: argument -f/--from: expected one argument
Error: Process completed with exit code 2.
```

### Expected Behavior

The `run_cdp_event_gather` command is called with the `JSON` configuration file which includes the default values and the prompt indicates the values are optional.

### Reproduction

Execute the Event Gather workflow manually without entering date values.

### Environment

_Any additional information about your environment._

-   OS Version: _[e.g. macOS 11.3.1]_
-   Cookiecutter Version: _[e.g. 0.5.0]_

--- a/{{ cookiecutter.hosting_github_repo_name }}/.github/workflows/event-gather-pipeline.yml
+++ b/{{ cookiecutter.hosting_github_repo_name }}/.github/workflows/event-gather-pipeline.yml
@@ -10,11 +10,9 @@ on:
       from:
         description: "Optional ISO formatted string for datetime to begin event gather from."
         required: false
-        default: ""  # Will get converted to N (default 2) days prior
       to:
         description: "Optional ISO formatted string for datetime to end event gather at."
         required: false
-        default: ""  # Will get converted to now
 
 permissions:
   id-token: write
@@ -102,6 +100,12 @@ jobs:
       if: {% raw %}${{ github.event_name == 'workflow_dispatch' }}{% endraw %}
       run: |
         cd python/
-        {% raw %}run_cdp_event_gather event-gather-config.json \
-          --from ${{ github.event.inputs.from }} \
-          --to ${{ github.event.inputs.to }}{% endraw %}
+        CDP_FROM_USER=${{ github.event.inputs.from }}
+        CDP_FROM_DEFAULT=$(date -Iseconds -d "2 days ago")
+        CDP_FROM=${CDP_FROM_USER:-$CDP_FROM_DEFAULT}
+        CDP_TO_USER=${{ github.event.inputs.to }}
+        CDP_TO_DEFAULT=$(date -Iseconds)
+        CDP_TO=${CDP_TO_USER:-$CDP_TO_DEFAULT}
+        run_cdp_event_gather event-gather-config.json \
+          --from $CDP_FROM \
+          --to $CDP_TO

--- a/{{ cookiecutter.hosting_github_repo_name }}/.github/workflows/event-gather-pipeline.yml
+++ b/{{ cookiecutter.hosting_github_repo_name }}/.github/workflows/event-gather-pipeline.yml
@@ -100,10 +100,10 @@ jobs:
       if: {% raw %}${{ github.event_name == 'workflow_dispatch' }}{% endraw %}
       run: |
         cd python/
-        CDP_FROM_USER=${{ github.event.inputs.from }}
+        CDP_FROM_USER={% raw %}${{ github.event.inputs.from }}{% endraw %}
         CDP_FROM_DEFAULT=$(date -Iseconds -d "2 days ago")
         CDP_FROM=${CDP_FROM_USER:-$CDP_FROM_DEFAULT}
-        CDP_TO_USER=${{ github.event.inputs.to }}
+        CDP_TO_USER={% raw %}${{ github.event.inputs.to }}{% endraw %}
         CDP_TO_DEFAULT=$(date -Iseconds)
         CDP_TO=${CDP_TO_USER:-$CDP_TO_DEFAULT}
         run_cdp_event_gather event-gather-config.json \

